Include/cpython/object.h

```diff
@@ -515,6 +515,8 @@ struct _ts;
 /* Python 3.9 private API, invoked by the macros below. */
 PyAPI_FUNC(int) _PyTrash_begin(struct _ts *tstate, PyObject *op);
 PyAPI_FUNC(void) _PyTrash_end(struct _ts *tstate);
+/* Python 3.10 private API, invoked by the macros below. */
+PyAPI_FUNC(int) _PyTrash_cond(void *op_raw, void *dealloc);
```

Comment:
I would prefer to pass an object to it.

```suggestion
PyAPI_FUNC(int) _PyTrash_cond(PyObject *op, void *dealloc);
```

And use `_PyObject_CAST(op)` in Py_TRASHCAN_BEGIN() macro.

Reply:
done.